README.md

```diff
@@ -1,8 +1,130 @@
 # expo-exposure-notification
-Expo module and test app for the Apple / Google Exposure Notification (aka Contact Tracing) APIs
 
+Expo module and test app for the Apple / Google Exposure Notification (fka Contact Tracing) APIs
 
-## Development usage
+
+## 🤔 Why?
+
+Apple and Google are working together on a standard Exposure Notification API that will across iOS and Android. The API will uses Bluetooth Low Energy (BLE) to implement a technique called contact tracing to track the spread of COVID-19 via approved apps from governments and health organizations.
+
+Developers have used Expo to build a number of COVID-19 symptom trackers because of the speed of deployment possible with Expo, so we want to make the E
+
```

Comment:
`...so we want to make the E`

accidental end?

Reply:
oops